core/server/api_container/server/startosis_engine/kurtosis_instruction/run_sh/run_sh.go

```diff
@@ -0,0 +1,237 @@
+package run_sh
```

Comment:
something that I suggest you: for every new instruction you have to add a new test case on the framework here: core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/test_engine

Reply:
will do this in the next PR @leoporoli . Thanks for the heads up.